autosklearn/data/abstract_data_manager.py

```diff
@@ -24,7 +24,7 @@ def perform_one_hot_encoding(
     rvals = []
     if any(categorical):
         encoder = DataPreprocessor(
-            categorical_features=categorical, force_sparse_output=sparse)
+            feat_type=categorical, force_sparse_output=sparse)
```

Comment:
We might actually be able to remove this function. I don't think it is used any more.

Reply:
Thanks, I have removed this